config/software/datadog-metro.rb

```diff
@@ -10,6 +10,11 @@
 
 dependency "libpcap"
 
+if ohai["platform_family"] == "rhel"
+  source :url => "https://s3.amazonaws.com/dd-agent/go-metro/gometro-centos6",
```

Comment:
Can you version the `gometro-centos6` binary? For instance `gometro-centos6-1.0.0` or whatever the current version is? Not sure the project is even properly versioned. There's actually a big PR I'd like to merge, maybe that could be `1.0.0`. Anyways, we probably want to do something like that.